src/css/clicker.css

```diff
@@ -220,6 +220,51 @@ body {
     transform: scale(1.05);
 }
 
+#save-button:active, #new-game-button:active {
+    transform: scale(0.95);
+}
+body {
```

Comment:
You already defined this styles in lines 8 - 13. Please remove all duplications.